### ISSUE TITLE
Centos7

### ISSUE THREAD

**Issue by Arnros** [ReinerNippes/nextcloud#17]
Hi

I'm installing your role to a new Centos 7.5 server.

I have changed path for ImageMagick : 
````
/etc/ImageMagick/policy.xml
````

I have installed manually  policycoreutils-python because it's missing : 
````
TASK [prep_nextcloud : selinux targets] *********************************************************************************************************************************************************************************************************************************************************************************************************************************************
Tuesday 13 November 2018  23:10:00 +0100 (0:00:00.416)       0:04:31.468 ******
failed: [localhost] (item=/var/nc-data(/.)?) => {"changed": false, "item": "/var/nc-data(/.)?", "msg": "This module requires policycoreutils-python"}
failed: [localhost] (item=/var/www/nextcloud/config(/.)?) => {"changed": false, "item": "/var/www/nextcloud/config(/.)?", "msg": "This module requires policycoreutils-python"}
failed: [localhost] (item=/var/www/nextcloud/apps(/.)?) => {"changed": false, "item": "/var/www/nextcloud/apps(/.)?", "msg": "This module requires policycoreutils-python"}
failed: [localhost] (item=/var/www/nextcloud/assets(/.)?) => {"changed": false, "item": "/var/www/nextcloud/assets(/.)?", "msg": "This module requires policycoreutils-python"}
failed: [localhost] (item=/var/www/nextcloud/.htaccess) => {"changed": false, "item": "/var/www/nextcloud/.htaccess", "msg": "This module requires policycoreutils-python"}
failed: [localhost] (item=/var/www/nextcloud/.user.ini) => {"changed": false, "item": "/var/www/nextcloud/.user.ini", "msg": "This module requires policycoreutils-python"}
````

**Comment by ReinerNippes** [comment]
Thanks. It's fixed with commit #9b82055

**Comment by Arnros** [comment]
Thanks
